internal/partinfo/ext_test.go

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) 2023 Canonical Ltd
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License version 3 as
+// published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package partinfo
```

Comment:
As I understand, you are suggesting I put the tests on a separate package (e.g. `partinfo_test`) and only use exported symbols. Am I wrong?

If that's the case, I don't really see the value in this particular example since I'm testing the internals of the superblock parser implementations (and thus doing black-box testing, for which using a separate package seems inconvenient).

I'm open to discuss this matter further (: